### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by Orzala-2024**
🎯 This is personal portfolio to showcase my skills and experienced . Here are some feathers in my portfolio:
-  It has a sleek layout  and responsive design for all sizes.
-  Viewers can contact me through any communication apps just by clicking related icons .
-  I have added all my projects images with their real links. Just by clicking on image they can  access  my project.
- I used keyframes for animations .

This repo include below files:
-index.html
-about.html
-contact.html
-projects.html
-styles.css
-background vedio
-5 images 

♦ I added media queries for these sizes :
@media (max-width: 1024px)
@media (max-width: 768px)
@media only screen and (max-width: 430px)
if it needs to add more media queries , kindly guide me to add which queries ?

### CODE REVIEW

**Comment by ahzamir** [comment]
@LimaSahibzada 

- I can not see your partner's feedback!

**Comment by Orzala-2024** [comment]
@ahzamir ,
as i mentioned in the group , my partner has told me that she is busy and won't check my project . so i submit my project. 
zuhra Hashimi is my partner.

**Comment by Orzala-2024** [comment]
@ahzamir , 
as i mentioned in the group , my partner has told me that she is busy and won't check my project . so i submit my project.
zuhra Hashimi is my partner.

**Comment by Orzala-2024** [comment]
@ahzamir ,
i just  added a media query .

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <div class="icon-container">
+          <a href="contact.html"><i class="fa fa-fw fa-envelope"></i></a>
+          <span class="icon-text">Contact</span>
+        </div>
+        <div class="icon-container">
+          <a href="#welcome-section-text"><i class="fa solid fa-file"></i></a>
+          <span class="icon-text">Projects</span>
+        </div>
+      </div>
+    </header>
```
Comment (ahzamir) [inline index.html]: - Remove the Target blank from your page links as it is internal page links you use target blank when you want to link some external websites.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          <span class="icon-text">About</span>
+        </div>
+        <div class="icon-container">
+          <a href="contact.html"><i class="fa fa-fw fa-envelope"></i></a>
+          <span class="icon-text">Contact</span>
+        </div>
+        <div class="icon-container">
+          <a href="#welcome-section-text"><i class="fa solid fa-file"></i></a>
+          <span class="icon-text">Projects</span>
+        </div>
```
Comment (ahzamir) [inline index.html]: Screenshot 2024-06-08 161111

- This one is not working

File: projects.html
Comment (ahzamir) [inline projects.html:1]: - Make some changes to this page mobile version design to make it align center and justify center:
Screenshot 2024-06-08 161018

**Review by ahzamir — CHANGES_REQUESTED** [review]
@LimaSahibzada 

- Well done lima you have done a great job implementing most of the requirements but still there are some issues on your project you need to work on
  - Chek the inline comments below
  
  - Add all of your subfiles like `about.html`, `contact.html`, `projects.html` inside a folder names /projects and then change the links to get the file from the project folder to make your project folder clean:
  
## Example:
```
  Portfolio/
  └── index.html
  └── pages/
    └── about.html
    └── projects.html
    └── contact.html
```

**Comment by Orzala-2024** [comment]
@ahzamir ,
I made changes in my project . kindly review it . 

thank you

**Pull request merged**